packages/tslint-rules/package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "@uifabric/tslint-rules",
   "version": "7.1.2",
-  "description": "Shared TSLint rules for UI Fabric projects",
+  "description": "Shared TSLint rules for Fluent UI React projects",
```

Comment:
is this specific to React?

Reply:
I think at the moment it is, because other existing variants don't use TS. If we add a TS-based variant which doesn't use React later, we can do a pass on updating all instances where appropriate.